MANIFEST.in

```diff
@@ -8,9 +8,11 @@ include c/enc/*.h
 include c/include/brotli/*.h
 include LICENSE
 include MANIFEST.in
-include python/_brotli.cc
+include python/_brotli.c
 include python/bro.py
 include python/brotli.py
+include python/brotli.pyi
+include python/py.typed
```

Comment:
I think these two files were moved inside python/brotli package directory

Reply:
Indeed.
I'm a bit confused why they still get included.
Looks like modern setuptools versions simply include pyi and py.typed themselves.

Reply:
done